oca_dependencies.txt

```diff
@@ -2,6 +2,3 @@ server-tools
 server-backend https://github.com/OCA/server-backend
```

Comment:
As this is a clean oca_dependency let us get rid of the direct pointing to OCA/server-backend

just leave `server-backend`